dms/models/directory.py

```diff
@@ -248,15 +249,17 @@ def check_access_token(self, access_token=False):
                 item = items[0]
                 if item.id == self.id:
                     return True
-                else:
-                    directory_item = self
-                    while directory_item.parent_id:
-                        if directory_item.id == item.id:
-                            return True
-                        directory_item = directory_item.parent_id
-                    # Fix last level
+
```

Comment:
Please don't add empty lines inside a method. It's less readable and requires more scroll while having syntax highlighting is enough. I don't repeat the comment more times, but the same for the entire modifications.